testing/py-traceback2/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-traceback2
```

Comment:
This module supports both Python 2 (legacy) and 3, so it must be packaged for Python 3 only (py3-traceback2), or for both Python 2 and 3 (py-traceback2 with subpackages py2-traceback2 and py3-traceback2). If there’s no specific reason for supporting Legacy Python, I’d prefer py3 only package (it’s simpler).

Reply:
any reason why this is still not pulled ???